Source/WebGPU/WGSL/Metal/MetalFunctionWriter.cpp

```diff
@@ -947,6 +948,7 @@ static ASCIILiteral convertToSampleMode(InterpolationType type, InterpolationSam
         }
     case InterpolationType::Perspective:
         switch (sampleType) {
+        default:
```

Comment:
Just a nit, but can we explicitly list the cases here?

Reply:
Yes, I'll make that change